### DISCUSSION TITLE
Access to community Slack

### DISCUSSION THREAD

**Discussion by janxbar** [rabbitmq/rabbitmq-server#9362] [Other]
Hi,
access to RabbitMQ community Slack seems to be limited to @pivotal.io and @rabbitmq.com emails yet the support page mentions it as one of the free support places. How can I join the community Slack without email on one of those domains? Or is the https://www.rabbitmq.com/contact.html out of date? 

Kind regards, Jan

**Comment by michaelklishin** [comment]
Slack's invitation system is broken and it affects OSS projects in curious ways every so often.

This is why we have set up a [Discord server](https://blog.rabbitmq.com/posts/2023/04/announcing-rabbitmq-community-discord-server/).
